app/controllers/books_controller.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class BooksController < ApplicationController
+  before_action :find_single_book, only: %i[show edit update destroy]
+
+  def index
+    @books = Book.all
+  rescue StandardError => e
```

Comment:
If we rescue `StandardError` here, all of the error under `StandardError` (which is pretty much every errors) will be rescued and just logged out in the rails server without being raised (so we don't know the error happened) We should only rescue the errors that we don't want it to be raised or want to handle it differently.

Reply:
Thx for comment krub :) 
Its mean that mostly errors will be logged in rails server which possible to be ignored, So it should rescues specific error which need to be rescued differently such as  `ZeroDivisionError` or `ArgumentError` etc.
I curious how our team handle error so i will follow this team practice krub 🙏